src/Symfony/Component/Security/Core/Authentication/RememberMe/CacheTokenVerifier.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Security\Core\Authentication\RememberMe;
+
+use Psr\Cache\CacheItemPoolInterface;
+
+/**
+ * @author Jordi Boggiano <j.boggiano@seld.be>
+ */
+class CacheTokenVerifier implements TokenVerifierInterface
+{
+    /** @var CacheItemPoolInterface */
+    private $cache;
+    /** @var int */
+    private $outdatedTokenTtl;
+    /** @var string */
+    private $cacheKeyPrefix;
```

Comment:
Please remove the PHPdoc from these properties.

In Symfony, we always skip the PHPdocs if they don't introduce anything that isn't already documented by the constructor (through typehints or PHPdoc)